.circleci/config.yml

```diff
@@ -46,7 +46,10 @@ jobs:
           command: python3 -m venv ~/curation_env && echo "source ~/curation_env/bin/activate" >> $BASH_ENV
       - run:
           name: Upgrade pip and install requirements
-          command: pip install --upgrade pip setuptools && pip install -r requirements.txt
+          command: |
+            pip install --upgrade pip setuptools
+            pip install -r requirements.txt
+            pip install -r deid/requirements.txt
```

Comment:
I think it's best if we include versions in `deid/requirements.txt`
However this doesn't need to be addressed now and it may not be as simple, since some package versions might conflict with other packages in `requirements.txt`. We need to do a `pip freeze` with everything working during our deid test run and add the right versions.